modules/external-auth-configuring.adoc

```diff
@@ -0,0 +1,178 @@
+// Module included in the following assemblies:
+//
+// * authentication/external-auth.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="external-auth-configuring_{context}"]
+= Configuring an external OIDC identity provider for direct authentication
+
+You can configure {product-title} to directly use an external OIDC identity provider to issue tokens for authentication.
+
+:FeatureName: Direct authentication with an OIDC identity provider
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* You have enabled the `TechnologyPreviewNoUpgrade` feature set.
```

Comment:
If `TechnologyPreviewNoUpgrade` should replace by `TechPreviewNoUpgrade`?

Reply:
Good catch! Updated